### PR TITLE
Display movie list

### PR DESCRIPTION
**Pull request by Kwentsir**
Created the main part of the homepage that keeps that displays the movies

### CODE REVIEW

File: src/modules/movieList.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const displayMovies = (data) => {
+  const displayMovies = document.querySelector('#movie-list');
+  data.forEach((movie) => {
+    displayMovies.innerHTML += `<article class="movie">
+<img src="${movie.image.medium}" alt="${movie.name}">
+<h2>${movie.name}</h2>
+<p class="genre">${movie.genres}</p>
+<div class="button">
+<button>Comments</button>
+<button>Reservations</button>
```
Comment (huseyinbiyik) [inline src/modules/movieList.js:10]: @Kwentsir you are doing a great job for the team! 🥇  There is no problem until now. It is optional but you can change the indention of the Template Strings to increase the readability of the code.
```suggestion
<img src="${movie.image.medium}" alt="${movie.name}">
<h2>${movie.name}</h2>
<p class="genre">${movie.genres}</p>
<div class="button">
<button>Comments</button>
<button>Reservations</button>
```
You can merge now! 👍

**Review by huseyinbiyik — APPROVED** [review]

**Review by SiansandaTamara — APPROVED** [review]
Great job !! time to merge it

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hello Joseph, sorry we approve it but there is an important change that needs to be done. Kindly change the button in this format
`button id=${movie.id}`

**Review by huseyinbiyik — APPROVED** [review]
Now, it looks like it is going to function. Thank you for the effort. You can merge it now! 👍

**Pull request merged**